### DISCUSSION TITLE
Ethernet connection drops after installing

### DISCUSSION THREAD

**Discussion by hubadubdubdub** [valinet/ExplorerPatcher#3026] [Q&A]
Hi guys. 

I don't see a submit a bug section, I'm sure I am overlooking one, but that might give an opportunity to ask for advice since I came to like Explorer Patcher over past half a year or so. 

The symptoms are a disconnect of all the mapped network drives upon install and an immediate reconnection or available access to the home media server upon uninstall. I haven't had the issue a few weeks ago, but recently that has been bugging me and I suspected my Synology, my firewall / firewall control, my vpn ...and (strangely) EP, but after "pushing all the buttons" and having the issue resolved (or it seemed like it) I installed EP again and the issue resurfaced immediately in front of me. 

To limit this to just one area: it concerns my home network / LAN and not WIFI or Internet, those are continuing to work fine.
And I had been running the latest update upon the issue and before uninstalling in the course of eliminating the source of the problem.

So, the question I would have is what could be causing this in regard to EP?
- settings?
- functionality / bug / recent update?

If it's settings, then community is probably best to resolve this. If the second one, then it needs to be addressed and it would be helpful if you'd provide me with a path to a way of submitting the bug.

Looking forward to be reading from you :)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

Can you put the list of steps that causes this?
